src/mozillavpn.cpp

```diff
@@ -1686,3 +1686,9 @@ void MozillaVPN::hardResetAndQuit() {
   hardReset();
   quit();
 }
+
+void MozillaVPN::crashTest() {
+  char* text = new char[100];
```

Comment:
Put it behind #ifdef MVPN_DEBUG

Reply:
This is for QA to test.  We don't have debug builds for them.  I'm not sure what putting it behind a feature means.